src/Plugin/Derivative/OgLocalTask.php

```diff
@@ -30,7 +30,7 @@ class OgLocalTask extends DeriverBase implements ContainerDeriverInterface {
    *
    * @var \Drupal\Core\Routing\RouteProvider
```

Comment:
The route provider service is replaced with a version that will rebuild (RouteProviderLazyBuilder) if routes used during installation. Not using an interface causes an error.
Also change the use statement at top of this page for this interface.
```suggestion
   * @var \Drupal\Core\Routing\RouteProviderInterface
```